config/initializers/assets.rb

```diff
@@ -0,0 +1 @@
+Rails.application.config.assets.precompile << %w(prx_auth-rails_manifest.js)
```

Comment:
What happens on rails apps without the asset pipeline?  (CMS / Feeder come to mind).

Reply:
Builds and boots fine.  I see that assets are disabled globally in e.g. cms, so this directive is a noop.